### PR TITLE
credential prompt

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added steps to address credential prompt issue

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 1]
```diff
@@ -93,6 +93,21 @@ Use the steps in [Troubleshooting mail enabled public folder synchronization fai
 
 ## Step 3: Configure Exchange Online users to access Exchange Server on-premises public folders
 
+A Microsoft 365 or Office 365 mailbox who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:96]:
```suggestion
A Microsoft 365 or Office 365 mailbox that is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.  
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 2]
```diff
@@ -93,6 +93,21 @@ Use the steps in [Troubleshooting mail enabled public folder synchronization fai
...
 
+A Microsoft 365 or Office 365 mailbox who is not represented by a MailUser object on-premises (local to the target public folder hierarchy) won't be able to access on-premises public folders.  
+
+You can use following command to identify such mailboxes:
+
+```PowerShell
+Get-Mailbox |?{$_.IsDirSynced -eq $false}
+```
+
+The above users will keep getting credential prompt after public folder mailboox access is configured. Use one of the following solution for such users, before enabling public folder access:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:104]:
```suggestion
These users will keep getting credential prompts after public folder mailbox access is configured. Use one of the following solutions for such users before enabling public folder access:
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 3]
```diff
@@ -93,6 +93,21 @@ Use the steps in [Troubleshooting mail enabled public folder synchronization fai
...
+
+1) Link the EXO only mailboxes, listed in step above, to On-Premises users by using the Knowledge Base article [Exchange Online users can't access legacy on-premises public folders](https://support.microsoft.com/help/3106618).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:106]:
```suggestion
1. Link the Exchange Online only mailboxes listed in the previous step to on-premises users by using the steps described in [Exchange Online users can't access legacy on-premises public folders](https://support.microsoft.com/help/3106618).
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 4]
```diff
@@ -93,6 +93,21 @@ Use the steps in [Troubleshooting mail enabled public folder synchronization fai
...
+
+2) Use steps provided in [Controlled Connections to Public Folders](https://aka.ms/ControlPF) to enable public folder access only to mailboxes that have linked users at On-Premises.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:108]:
```suggestion
2. Use the steps provided in [Controlled Connections to Public Folders](https://aka.ms/ControlPF) to enable public folder access only to mailboxes that have linked users on-premises.
```

File: Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md [hunk 5]
```diff
@@ -93,6 +93,21 @@ Use the steps in [Troubleshooting mail enabled public folder synchronization fai
...
+
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/set-up-modern-hybrid-public-folders.md:109]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — COMMENTED** [review]
@Batre-MSFT - Please also commit the changes to lines 96, 106, 108, and 109. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**